### ISSUE TITLE
Activate sentry

### ISSUE THREAD

**Issue by Siggi0904** [MiRo1310/ioBroker.alexa-timer-vis#141]
Hi,

it's more an option feature request.
Can you activate sentry in any future version, if it better for you to see logs?

Thanks.

**Comment by MiRo1310** [comment]
Yes, it would certainly be possible to use sentry in the future.
